src/main/java/org/prebid/server/metric/Metrics.java

```diff
@@ -164,6 +164,12 @@ public void updateAccountRequestMetrics(String accountId, MetricName requestType
         }
     }
 
+    public void increaseAccountRejectedRequestCounter(String accountId) {
```

Comment:
Pls rename to `updateAccountRequestRejectedMetrics` to be in one style with other method names.